tests/unit/mainwindow/test_prompt.py

```diff
@@ -23,8 +23,15 @@
 from PyQt5.QtCore import Qt
 
 from qutebrowser.mainwindow import prompt as promptmod
-from qutebrowser.utils import usertypes
-
+from qutebrowser.utils import usertypes, objreg
+from qutebrowser.misc import cmdhistory
+
+@pytest.fixture(autouse=True)
+def test_init(fake_save_manager, data_tmpdir, config_stub):
```

Comment:
This isn't a test, so it shouldn't be named `test_init`. You could name it `cmdhistory_init` or so.